### DISCUSSION TITLE
it seems like installation gets stuck on a blank screen

### DISCUSSION THREAD

**Discussion by iqworks** [Security-Onion-Solutions/securityonion#9371] [General]
Hi all,  
I am using :
    Dell XPS 17 9720
    SO version : securityonion-2.3.190-20221207.
    I am trying to use SO on my windows 11 laptop, I don’t use the cloud.
    I used securityonion-2.3.190-20221207.iso
    I used Centos 7
    For nodes, I just have laptops, TV’s and a printer.
    I am using a dell laptop and windows 11 version 10.0.22621 build 19044

    At the beginning of the installation, I enter all of the initial information, it then begins (installing centos 7) to load the docker containers etc. 

    At some point, the screen goes blank. It never shows the “initial install complete. Press [enter] to reboot!”. I thought maybe it was still processing. The next morning it was still on a blank screen. At this point I powered off. When I powered on, I got the “welcome to security onion install” ?

**Comment by dougburks** [comment]
You previously reported similar behavior and here was my response:
https://github.com/Security-Onion-Solutions/securityonion/discussions/8907#discussioncomment-3914358

I'd recommend checking those same things again.

**Reply by iqworks** [reply]
Thanks again Doug, i will review that post
